spec/factories.rb

```diff
@@ -1,4 +1,12 @@
 FactoryBot.define do
+
+  factory :user do
+    email {"test@gmail.com"}
```

Comment:
Did you check if you were following the style guide? https://github.com/airbnb/ruby#spaces-operators